src/components/DataTable/TableToolbarSearch.js

```diff
@@ -34,6 +35,7 @@ TableToolbarSearch.propTypes = {
   /**
    * Provide an optional class name for the search container
    */
+  id: PropTypes.string,
```

Comment:
Please add a comment for the new `id` prop here so that the prop definitions and the comments are in sync.

Reply:
Also, is this a prop type definition that needs to be required, or are we providing a fallback value?